### PR TITLE
Create product-docs-may.md

### PR DESCRIPTION
**Pull request by runleonarun**
## What are you changing in this pull request and why?

Adding May updates to Product Docs.

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `website/static/_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/product-docs-may.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+title: "May 2023 product docs updates"
+id: "May-product-docs"
+description: "May 2023: Find out what the product docs team has been busy doing in the month of May."
+sidebar_label: "Update: product docs changes"
+sidebar_position: 8
```
Comment (runleonarun) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/product-docs-may.md]:
```suggestion
sidebar_position: 1
```

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/product-docs-may.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+## ☁ Cloud projects
+
+- We launched [dbt Cloud IDE user interface doc](/docs/cloud/dbt-cloud-ide/ide-user-interface), which provides a thorough walkthrough of the IDE UI elements and their definitions.
+- Launched a sparkling new [dbt Cloud Scheduler page](/docs/deploy/job-scheduler) ✨! We went from previously having little content around the scheduler to a subsection that breaks down the awesome scheduler features and how it works.
+- Updated the [dbt Cloud user license page](/docs/cloud/manage-access/seats-and-users#licenses) to clarify how to add or remove cloud users.
+- Shipped these Discovery API docs to coincide with the launch of the Discovery API:
+    - [About the Discovery API](/docs/dbt-cloud-apis/discovery-api)
+    - [Use cases and examples for the Discovery API](/docs/dbt-cloud-apis/discovery-use-cases-and-examples)
+    - [Query the Discovery API](/docs/dbt-cloud-apis/discovery-querying)
+    - [Project state in dbt Cloud](/docs/deploy/project-state#project-state-in-dbt-cloud)
```
Comment (runleonarun) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/product-docs-may.md]:
```suggestion
```

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/product-docs-may.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
+---
+title: "May 2023 product docs updates"
+id: "May-product-docs"
+description: "May 2023: Find out what the product docs team has been busy doing in the month of May."
+sidebar_label: "Update: product docs changes"
+sidebar_position: 1
+tags: [May-2023,product-docs]
+---
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/product-docs-may.md:9]:
```suggestion
date: 06-01-2023
---
```

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/product-docs-may.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+- Updated the [dbt Cloud user license page](/docs/cloud/manage-access/seats-and-users#licenses) to clarify how to add or remove cloud users.
+- Shipped these Discovery API docs to coincide with the launch of the Discovery API:
+    - [About the Discovery API](/docs/dbt-cloud-apis/discovery-api)
+    - [Use cases and examples for the Discovery API](/docs/dbt-cloud-apis/discovery-use-cases-and-examples)
+    - [Query the Discovery API](/docs/dbt-cloud-apis/discovery-querying)
+
+## 🎯 Core projects
+
+- See what’s coming up [in Core v 1.6](https://github.com/dbt-labs/docs.getdbt.com/issues?q=is%3Aissue+label%3A%22dbt-core+v1.6%22)!
+- We turned the `profiles.yml` [page](/reference/profiles.yml) into a landing page, added more context to profile.yml page, and moved the ‘About CLI’ higher up in the `Set up dbt` section.
```
Comment (runleonarun) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/product-docs-may.md]:
```suggestion
- We turned the `profiles.yml` [page](/docs/core/connect-data-platform/profiles.yml) into a landing page, added more context to profile.yml page, and moved the ‘About CLI’ higher up in the `Set up dbt` section.
```

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/product-docs-may.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
+---
+title: "May 2023 product docs updates"
+id: "May-product-docs"
+description: "May 2023: Find out what the product docs team has been busy doing in the month of May."
+sidebar_label: "Update: product docs changes"
+sidebar_position: 1
+tags: [May-2023,product-docs]
+date: 06-01-2023
```
Comment (runleonarun) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/product-docs-may.md]:
```suggestion
date: 2023-06-01
```

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/product-docs-may.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+tags: [May-2023,product-docs]
+date: 2023-06-01
+---
+
+
+## May **2023 product docs updates**
+
+Hello from the dbt Docs team: @mirnawong1, @matthewshaver, @nghi-ly, and @runleonarun! First, we’d like to thank the 13 new community contributors to docs.getdbt.com!
+
+What's new to [docs.getdbt.com](http://docs.getdbt.com/) in May:
```
Comment (runleonarun) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/product-docs-may.md]:
```suggestion
Here's what's new to [docs.getdbt.com](http://docs.getdbt.com/) in May:
```

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/product-docs-may.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+## 🎯 Core projects
+
+- See what’s coming up [in Core v 1.6](https://github.com/dbt-labs/docs.getdbt.com/issues?q=is%3Aissue+label%3A%22dbt-core+v1.6%22)!
+- We turned the `profiles.yml` [page](/docs/core/connect-data-platform/profiles.yml) into a landing page, added more context to profile.yml page, and moved the ‘About CLI’ higher up in the `Set up dbt` section.
+
+## New 📚 Guides, ✏️ blog posts, and FAQs
+
+If you want to contribute to a blog post, we’re focusing on content
+
+- **[Accelerate your documentation workflow: Generate docs for whole folders at once](/blog/generating-dynamic-docs-dbt)**
```
Comment (runleonarun) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/product-docs-may.md]:
```suggestion
- Published a blog post: [Accelerate your documentation workflow: Generate docs for whole folders at once](/blog/generating-dynamic-docs-dbt)
```

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/product-docs-may.md [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+- **[Data engineers + dbt v1.5: Evolving the craft for scale](/blog/evolving-data-engineer-craft)**
```
Comment (runleonarun) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/product-docs-may.md]:
```suggestion
- Published a blog post: [Data engineers + dbt v1.5: Evolving the craft for scale](/blog/evolving-data-engineer-craft)
```

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/product-docs-may.md [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+- We turned the `profiles.yml` [page](/docs/core/connect-data-platform/profiles.yml) into a landing page, added more context to profile.yml page, and moved the ‘About CLI’ higher up in the `Set up dbt` section.
+
+## New 📚 Guides, ✏️ blog posts, and FAQs
+
+If you want to contribute to a blog post, we’re focusing on content
+
+- Published a blog post: [Accelerate your documentation workflow: Generate docs for whole folders at once](/blog/generating-dynamic-docs-dbt)
+- Published a blog post: [Data engineers + dbt v1.5: Evolving the craft for scale](/blog/evolving-data-engineer-craft)
+- Added an [FAQ](/faqs/Warehouse/db-connection-dbt-compile) to clarify the common question users have of *Why does dbt compile need to connect to the database?*
+- Published a [discourse article](https://discourse.getdbt.com/t/how-to-configure-external-user-email-notifications-in-dbt-cloud/8393) about configure job notifications for non-dbt Cloud users
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/product-docs-may.md]:
```suggestion
- Published a [discourse article](https://discourse.getdbt.com/t/how-to-configure-external-user-email-notifications-in-dbt-cloud/8393) about configuring job notifications for non-dbt Cloud users
```

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/product-docs-may.md [hunk 10]
```diff
@@ -0,0 +1,46 @@
...
+- See what’s coming up [in Core v 1.6](https://github.com/dbt-labs/docs.getdbt.com/issues?q=is%3Aissue+label%3A%22dbt-core+v1.6%22)!
+- We turned the `profiles.yml` [page](/docs/core/connect-data-platform/profiles.yml) into a landing page, added more context to profile.yml page, and moved the ‘About CLI’ higher up in the `Set up dbt` section.
+
+## New 📚 Guides, ✏️ blog posts, and FAQs
+
+If you want to contribute to a blog post, we’re focusing on content
+
+- Published a blog post: [Accelerate your documentation workflow: Generate docs for whole folders at once](/blog/generating-dynamic-docs-dbt)
+- Published a blog post: [Data engineers + dbt v1.5: Evolving the craft for scale](/blog/evolving-data-engineer-craft)
+- Added an [FAQ](/faqs/Warehouse/db-connection-dbt-compile) to clarify the common question users have of *Why does dbt compile need to connect to the database?*
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/product-docs-may.md]:
```suggestion
- Added an [FAQ](/faqs/Warehouse/db-connection-dbt-compile) to clarify the common question users have on *Why does dbt compile needs to connect to the database?*
```

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/product-docs-may.md [hunk 11]
```diff
@@ -0,0 +1,46 @@
+---
+title: "May 2023 product docs updates"
+id: "May-product-docs"
+description: "May 2023: Find out what the product docs team has been busy doing in the month of May."
+sidebar_label: "Update: product docs changes"
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/product-docs-may.md]:
```suggestion
sidebar_label: "Update: Product docs changes"
```

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/product-docs-may.md [hunk 12]
```diff
@@ -0,0 +1,46 @@
...
+sidebar_position: 1
+tags: [May-2023,product-docs]
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/product-docs-may.md]:
```suggestion
tags: [May-2023, product-docs]
```

**Review by mirnawong1 — APPROVED** [review]
hey @runleonarun , small nits but it looks good to me! how exciting!

**Pull request merged**